app/backend/wells/models.py

```diff
@@ -1285,14 +1286,15 @@ class Screen(AuditModel):
     """
     screen_guid = models.UUIDField(
         primary_key=True, default=uuid.uuid4, editable=False)
-    activity_submission = models.ForeignKey(
-        ActivitySubmission, db_column='filing_number', on_delete=models.CASCADE, blank=True, null=True)
-    well = models.ForeignKey(Well, db_column='well_tag_number',
-                             on_delete=models.CASCADE, blank=True, null=True)
-    screen_from = models.DecimalField(max_digits=7, decimal_places=2, verbose_name='From',
-                                      blank=True, null=True, validators=[MinValueValidator(Decimal('0.00'))])
-    screen_to = models.DecimalField(max_digits=7, decimal_places=2, verbose_name='To',
-                                    blank=False, null=True, validators=[MinValueValidator(Decimal('0.01'))])
+    activity_submission = models.ForeignKey(ActivitySubmission, db_column='filing_number',
+                                            on_delete=models.CASCADE, blank=True, null=True,
+                                            related_name='screen_set')
+    well = models.ForeignKey(Well, db_column='well_tag_number', on_delete=models.CASCADE, blank=True,
+                             null=True, related_name='screen_set')
+    start = models.DecimalField(db_column='screen_from', max_digits=7, decimal_places=2, verbose_name='From',
+                                blank=True, null=True, validators=[MinValueValidator(Decimal('0.00'))])
+    end = models.DecimalField(db_column='screen_to', max_digits=7, decimal_places=2, verbose_name='To',
+                              blank=False, null=True, validators=[MinValueValidator(Decimal('0.01'))])
```

Comment:
Not in your scope for this PR but this `validators=` code is a bit odd.  There must be a better way to handle "the end value should be more than the start value`

Reply:
Yeah, I guess we'll look at it when we do validation.